operator/cmd/mesh/uninstall.go

```diff
@@ -200,6 +200,21 @@ func uninstall(cmd *cobra.Command, ctx cli.Context, rootArgs *RootArgs, uiArgs *
 	if err := h.DeleteControlPlaneByManifests(manifestMap, iop.Spec.Revision, uiArgs.purge); err != nil {
 		return fmt.Errorf("failed to delete control plane by manifests: %v", err)
 	}
+	iopName := savedIOPName(iop)
```

Comment:
```	
iopName := savedIOPName(iop)
```

Relying on this is incompatible with https://github.com/istio/istio/pull/47860, I think?

Reply:
Yes this is incompatible. I wasn't excepting #47860 when I was trying to fix the current problem, but that is much desired solution. I'm going to close this one.